frontend/test/metabase/scenarios/dashboard-filters/reproductions/13150.cy.spec.js

```diff
@@ -0,0 +1,84 @@
+import { restore, expectedRouteCalls } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATASET;
+
+const questionDetails = {
+  name: "13150 (Products)",
+  query: { "source-table": PRODUCTS_ID },
+};
+
+const parameters = [
+  { name: "Title", slug: "title", id: "9f20a0d5", type: "category" },
+  {
+    name: "Category",
+    slug: "category",
+    id: "719fe1c2",
+    type: "category",
+  },
+  { name: "Vendor", slug: "vendor", id: "a73b7c9", type: "category" },
+];
+
+const [titleFilter, categoryFilter, vendorFilter] = parameters;
+
+describe("issue 13150", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route("POST", "/api/card/*/query").as("cardQuery");
+
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should not send additional card queries for all filters (metabase#13150)", () => {
+    cy.createQuestionAndDashboard({ questionDetails }).then(
+      ({ body: { id, card_id, dashboard_id } }) => {
+        cy.log("Add 3 filters to the dashboard");
+        cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
+          parameters,
+        });
+
+        cy.log("Add previously created qeustion to the dashboard");
+
+        cy.log("Connect all filters to the card");
+        cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
+          cards: [
```

Comment:
I'd also consider extracting this to a function.

It'd have 5 arguments if I am not mistaken, which is a bit too much. 
All in all this test might still be easier to step into.

Reply:
I hear you.
https://github.com/metabase/metabase/pull/16970#discussion_r667336235

These things take time, but are not forgotten.